### PR TITLE
Feature: Users can now assign multiple products to a white glove serv…

### PR DESCRIPTION
**Pull request by OniiCoder**
Users  can now assign multiple products to a given white glove service. These products would be used in the cart to match with the cart items to make sure at least one of the products exists.

### CODE REVIEW

File: resources/views/models/product-variants/form.blade.php [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                title="Product Array 1"
+                description="Link products to this parent product"
+                class="sh-mt-4"
+                allow-overflow
+                x-on:fab-added="$wire.call('selectProduct', $event.detail[1].key, 1)"
+                x-on:fab-removed="$wire.call('unselectProduct', $event.detail[1].key, 1)"
+            >
+                @if($model->exists)
+                    <x-fab::forms.combobox
+                        :componentKey="444444"
```
Comment (iamkarsoft) [inline resources/views/models/product-variants/form.blade.php:49]: @OniiCoder Awesome job on this PR. Care to explain why `componentKey="4444"`. Thanks 😄

File: src/Models/GoLoadUpProduct.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    public function parentProduct()
+    {
+        return $this->belongsTo(ProductVariant::class, 'parent_product_variant_id');
+    }
+
+    public function products()
+    {
+        return $this->belongsToMany(Product::class, 'go_load_up_shopify_product', )
```
Comment (iamkarsoft) [inline src/Models/GoLoadUpProduct.php]: I think you left a trailing `,` here in this products's function. @OniiCoder

File: resources/views/models/product-variants/form.blade.php (hunk 1 above)
Comment (garos4) [reply resources/views/models/product-variants/form.blade.php:49]: lol @iamkarsoft I think it unique key for that component and he decided to use 4444...maybe its his lucky number😂
I can also see componentKey='33' in the next component

Comment (OniiCoder) [reply resources/views/models/product-variants/form.blade.php:49]: 😂😂😂😂😂😂 Well, it was a random unique number that came to my mind.

File: src/Models/GoLoadUpProduct.php (hunk 2 above)
Comment (OniiCoder) [reply src/Models/GoLoadUpProduct.php]: Thanks for the catch.

**Review by mcespo — APPROVED** [review]

**Review by mcespo — APPROVED** [review]
HOLY shii - this is an impressive app no matter how I shake it. Working with it in my local env has been a treat. I cannot express how incredible this app is. Some elements here feel curious such as the usage of numbers to illustrate code combinations - though I probably would have eventually done something similar.

Thank you for all your work 🍾  🥳

**Pull request closed without merging**